components/QuestChain/QuestChainV1Page.tsx

```diff
@@ -65,6 +66,17 @@ import { UploadImageForm } from '../UploadImageForm';
 import { QuestsEditor } from './QuestsEditor';
 import { RolesEditor } from './RolesEditor';
 
+export const PoHAPI =
```

Comment:
can we move these constants to a separate utils file?

Reply:
done